devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
What is the reasoning here? The Envoy updates once every 5 minutes so that should be the maximum poll interval in order to get complete data. I chose 2 minutes since the parse function doesn't seem to be called reliably for every `pullData()` call (unless this is just another logging bug in the platform…) so that I typically get almost every new data set.